exporters/trace/gcp_exporter/internal/gcp_exporter_benchmark.cc

```diff
@@ -0,0 +1,229 @@
+#include <benchmark/benchmark.h>
+#include "exporters/trace/gcp_exporter/gcp_exporter.h"
+#include "opentelemetry/nostd/span.h"
+
+namespace cloudtrace_v2 = google::devtools::cloudtrace::v2;
+
+// These constants affect the overall runtime of the benchmark tests
+constexpr int kNumSpans = 100;
+constexpr int kNumIntAttributes = 30;
```

Comment:
Nit: You could consolidate these into one `kNumAttributes` variable

Reply:
I initially did that but I wanted to handle each of the attribute types differently.
In the future other people who pick up on this could handle each one separately too.